pitop/processing/algorithms/line_detect.py

```diff
@@ -1,13 +1,15 @@
-from numpy import arctan, array, pi
-
+from numpy import array
```

Comment:
Nothing mentioned about line following in the PR description. Please amend the description to provide a complete description of the code changes proposed here!

Reply:
There is one bullet point mentioning it, this is just cleaning up based on what has been developed since line_detect was made